src/js/lib/jquery.kazoosdk.js

```diff
@@ -588,6 +588,9 @@
 				'update': { 'verb': 'POST', 'url': 'accounts/{accountId}/match_lists/{matchListId}' },
 				'patch': { 'verb': 'PATCH', 'url': 'accounts/{accountId}/match_lists/{matchListId}' },
 				'delete': { 'verb': 'DELETE', 'url': 'accounts/{accountId}/match_lists/{matchListId}' }
+			},
+			configs: {
+				'get': { 'verb': 'GET', 'url': 'accounts/{accountId}/configs/kazoo_endpoint'}
```

Comment:
I think that kazoo_endpoint should be a parameter, because I guess that this endpoint may also be used for other types of configs.